ocs_ci/utility/utils.py

```diff
@@ -1997,6 +1997,34 @@ def destroy_cluster(installer, cluster_path, log_level="DEBUG"):
         log.error(traceback.format_exc())
 
 
+def config_to_string(config):
+    """
+    Convert ConfigParser object to string in INI format.
+
+    Args:
+        config (obj): ConfigParser object
+
+    Returns:
+        str: Config in one string
+
+    """
+    string = ""
+    sections = [config.default_section] + config.sections()
+
+    first = True
+    for section in sections:
+        if config.items(section):
+            if first:
+                first = False
+            else:
+                string += "\n\n"
+            string += f"[{section}]\n"
+            string += "\n".join(
+                [f"{item[0]}={item[1]}" for item in config.items(section)]
+            )
+    return string
```

Comment:
Are you implementing this because configparser object does allow to write string representation to file object only? If yes, it's would be better to just use `io.StringIO` object in `configparser.write(fileobject)` call instead of re-implementing string dump of configparser ini file.

Something like (this is just a draft, it's not a finalized code :):

```
strio = io.StringIO()
cp.write(strio)
string = strio.getvalue()
```